scripts/run_bq_pg_deep_compare.sh

```diff
@@ -2,28 +2,24 @@
 
 set -e
 
+source ./conf/config.pg
 source ./conf/config.bq
 
 DATE=$(date +"%s")
 CHECK=$((DATE / 86400 % 5 ))
 if [ $CHECK -ne 3 ]; then
     echo "Not an epoch boundary. Exiting..."
-    exit 0
+#    exit 0
 else
     echo "Epoch boundary, running BQ/PG deep comparison"
 fi
 
-export PGPASSWORD=$(jq -r .password <<< "$DB_CONFIG")
-export PGHOST=$(jq -r .host <<< "$DB_CONFIG")
-export PGDATABASE=$(jq -r .dbname <<< "$DB_CONFIG")
-export PGDATABASE_TESTNET=$(jq -r .dbname_test <<< "$DB_CONFIG")
-export PGPORT=$(jq -r .port <<< "$DB_CONFIG")
-export PGUSER=$(jq -r .username <<< "$DB_CONFIG")
-
 export BQUSER=$(jq -r .client_email <<< "$BQ_CONFIG")
 echo $BQ_CONFIG > ./key.json
 
-EPOCH_NO=$1
+#EPOCH_NO=$1
+# calculate last full epoch number from time difference to start of mainchain
+EPOCH_NO=$(( ($(date '+%s') - 1506203091) / 3600 / 24 / 5 - 1))
```

Comment:
The epoch being passed as an optional parameter, was that so that we can run the deep comparison manually and select the epoch we want (not necessarily the latest one).
In this line:
https://github.com/Blockchain-Data-Analytics/Cardano_on_BigQuery/blob/released/scripts/deep_compare/bq_pg_deep_compare.py#L113
the python script checks if there is such a parameter and if not retrieves the latest epoch from the db.
So maybe we could revert this?